tests/e2e/specs/NavTest.spec.js

```diff
@@ -0,0 +1,20 @@
+// https://docs.cypress.io/api/introduction/api.html
+
+describe("Test navigation", () => {
+  it("Navigation should exist", () => {
+    cy.visit("/");
+    cy.get("nav").should("exist");
+  });
+  it("Navigation should be clickable", () => {
+    cy.get("[data-cy='nav-elements']").click({ multiple: true });
+  });
+  it("Navigation elements should be focusable", () => {
+    // cypress can't test focus on these elements
+  });
+  it("Navigation should go to correct URL", () => {
```

Comment:
This doesn't test the navigation. Just activates the routing configuration.
- Is there a possibility that this test could fail? 
- `cy.visit` is using the "address input" of a browser to navigate to the URL. Nothing in the app.

Reply:
completely right, we have addressed the specific elements, now it makes much more sense